src/popup/router/pages/More.vue

```diff
@@ -67,12 +101,84 @@ export default {
   margin: 0 auto;
 
   .navigation {
-    padding: 12px;
-    display: flex;
-    flex-wrap: wrap;
+    width: 100%;
 
     .box-button {
```

Comment:
I suggest not to use class names that does not exist in the current component as we can't be 100% sure that someone else will remove/replace it in the future. And in general makes the code much harder to refactor.